src/modules/main/sidebar/portal/Portal.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+
+import { NavLink } from 'react-router-dom';
+
+import { Button } from '@material-ui/core';
+import Cancel from '@material-ui/icons/Cancel';
+
+import csx from './Portal.scss'
+import csx2 from '../Sidebar.scss'
```

Comment:
Components can only use one style file. If you are using 2 - you have badly planned components

Reply:
Has been fixed